### ISSUE TITLE
Request: Add musl/static builds to release assets, for easy installation on Alpine, etc.

### ISSUE THREAD

**Issue by AndydeCleyre** [rami3l/pacaptr#4]
On Alpine, the prebuilt amd64 asset:

```console
$ ldd /usr/local/bin/pacaptr
        /lib64/ld-linux-x86-64.so.2 (0x7f695f48c000)
        libdl.so.2 => /lib64/ld-linux-x86-64.so.2 (0x7f695f48c000)
        librt.so.1 => /lib64/ld-linux-x86-64.so.2 (0x7f695f48c000)
        libpthread.so.0 => /lib64/ld-linux-x86-64.so.2 (0x7f695f48c000)
Error loading shared library libgcc_s.so.1: No such file or directory (needed by /usr/local/bin/pacaptr)
        libc.so.6 => /lib64/ld-linux-x86-64.so.2 (0x7f695f48c000)
Error loading shared library ld-linux-x86-64.so.2: No such file or directory (needed by /usr/local/bin/pacaptr)
Error relocating /usr/local/bin/pacaptr: _Unwind_Resume: symbol not found
Error relocating /usr/local/bin/pacaptr: _Unwind_GetRegionStart: symbol not found
Error relocating /usr/local/bin/pacaptr: _Unwind_SetGR: symbol not found
Error relocating /usr/local/bin/pacaptr: _Unwind_GetDataRelBase: symbol not found
Error relocating /usr/local/bin/pacaptr: _Unwind_GetLanguageSpecificData: symbol not found
Error relocating /usr/local/bin/pacaptr: _Unwind_RaiseException: symbol not found
Error relocating /usr/local/bin/pacaptr: _Unwind_FindEnclosingFunction: symbol not found
Error relocating /usr/local/bin/pacaptr: _Unwind_GetIP: symbol not found
Error relocating /usr/local/bin/pacaptr: _Unwind_Backtrace: symbol not found
Error relocating /usr/local/bin/pacaptr: _Unwind_GetIPInfo: symbol not found
Error relocating /usr/local/bin/pacaptr: _Unwind_GetTextRelBase: symbol not found
Error relocating /usr/local/bin/pacaptr: _Unwind_SetIP: symbol not found
```

**Comment by rami3l** [comment]
@AndydeCleyre Acknowledged!
Sorry for the inconvenience, but my finals are coming and I might be slow to respond. :(

**Comment by rami3l** [comment]
@AndydeCleyre I'm not familiar with this, but according to [this issue](https://github.com/rust-lang/rust/pull/58575), I think when the target is musl, the output binary should be fully static.
However I met this problem during the test for Alpine (you can see my [test action](https://github.com/rami3l/pacaptr/blob/master/.github/workflows/test.yml) for this repo). It seems that simply targeting musl won't work unless a dynamic binary output is specified. 🤔

**Comment by rami3l** [comment]
@AndydeCleyre  Interesting issue...
It seems that I must cross-compile it using Ubuntu and musl target... (See [here](https://github.com/rust-lang/rust/issues/40174#issuecomment-538791091_).)

**Comment by rami3l** [comment]
@AndydeCleyre I gave it a try in 5a366e3. Not sure if it's working or not, but I think this must be a good start (just tried to run `pacaptr -h` in my Docker container and it seems to be fine). 

By the way, it seems that these two `linux` binaries are not compatible (the one built on ubuntu using `gnu` toolchain works fine with fedora, for example, but not for alpine, and I believe the same is true in another direction)... How may I distribute these binaries in a possibly more elegant way?

**Comment by rami3l** [comment]
@AndydeCleyre Alright. This issue should be closed by f38cf01: I've replaced the original linux binary with a static musl binary.

Please feel free to comment if you encounter any problem.
